### ISSUE TITLE
🚀 Feature: Use of theme (MUI) and styled component for styling

### ISSUE THREAD

**Issue by sudhanshutech** [GDSCITM/GDSCITM-official-website#134]
### 📜 Feature Description

Currently we are using inline style for styling so much which is not a good practice. We need to use themes property from mui using styled component.

### 🎤 Pitch

Create a style folder and under style folder create themes directory and using import ` ./style/themes` assign theme/colors to the respective container,buttons or text. Use styled component for styling if there mui colors are not used

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🔖 Have you read the Code of Conduct?

- [X] Yes, I have read Code of Conduct.

**Comment by HemanthPaila** [comment]
Hey, @sudhanshutech I understand the problem. I will replace the inline CSS with the external CSS. I am interested to solve this. Could you please assign It to me?

**Comment by sudhanshutech** [comment]
That's great to know @HemanthPaila , go ahead

**Comment by Yashsharma1911** [comment]
@HemanthPaila you are constantly contributing to GDSC thinking to give you gdsc badge 😉

**Comment by HemanthPaila** [comment]
Yah @Yashsharma1911. I am looking forward to solving bit more complex issues like routing setup, Redux (app-wide state management), and anything related to development. Thanks a lot.
- @Yashsharma1911 do you have any open-source community for me to join?

**Comment by Yashsharma1911** [comment]
> Yah @Yashsharma1911. I am looking forward to solving bit more complex issues like routing setup, Redux (app-wide state management), and anything related to development. Thanks a lot.
> 
> * @Yashsharma1911 do you have any open-source community for me to join?

Yes we will send you org invitation @ArunPatel02 please send org invitation to @HemanthPaila he is doing good work here we should appreciate it with GDSC tag in his profile 😎
